environment_py3_osx.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - python>=3.4
+  - python=3.8
```

Comment:
do you want to leave it in ? I mean, you tested it works now for 3.8, but shouldn't >=3.4 be the correct dependency ?

Reply:
Yes, this can be reverted now. That means we don't check for python3.8 explicitly, but at least now we know that there are no more errors with the present code